agent/command.go

```diff
@@ -120,8 +120,14 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 				}
 			}
 		case <-ctx.Done():
-			tc.logger.Task().Errorf("Command canceled: %v", err)
-			return errors.Wrap(err, "command canceled")
+			switch ctx.Err() {
+			case context.DeadlineExceeded:
+				deadline, _ := ctx.Deadline()
+				tc.logger.Task().Errorf("Command timed out: %v", deadline)
```

Comment:
Gotcha, maybe no reason to case on error then.

Reply:
I don't think that's what Brian meant - ctx.Deadline() doesn't return how long the timeout was for so you should omit that from the message. The rest is fine

Reply:
I think that one consistent error format is better since the context errors themselves are sufficient explanations, but I can change it back if we prefer the more verbose descriptions.